loopback4/.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "@contrast",
+  "rules": {
+    "no-console": "off"
```

Comment:
very small, but why is this here? if you need to override on a line you can do 
`// eslint-disable-line no-console`